src/mps/mps.jl

```diff
@@ -144,15 +144,28 @@ function randomCircuitMPS(
 
   l = Vector{Index}(undef, N)
 
+  # Compute the bond dimension for each link.
```

Comment:
Would be nice to have this as a separate function, so we could use it other places in the code and simplify the code here.

Maybe `linkdims_upper_bound`? It doesn't have to be exported, just internal use for now.